src/main/java/mainpack/MainRun.java

```diff
@@ -24,82 +24,69 @@ public void incomeMenu(){
     public static void main(String[] args) {
         MainRun menu = new MainRun();
         Database database = new Database();
+        InOutHndler handler = new InOutHndler();
```

Comment:
Up to you, but you can have several different handlers at the time, I'd rather renamed this variable to inOutHandler